### PR TITLE
Created errata document

### PR DESCRIPTION
**Pull request by andrea-perego**
Done by following the instructions at https://github.com/w3c/display_errata/ , as discussed in https://github.com/w3c/dxwg/issues/1182

Preview:

https://raw.githack.com/w3c/dxwg/andrea-perego-dxwg-errata/errata/index.html

### CODE REVIEW

File: errata/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  * The community discusses the issue. If it is accepted as a genuine erratum, the label “`Errata`” is added to the entry and the “`ErratumRaised`” label should be removed. Additionally, a new comment on the issue MAY be added, beginning with the word "Summary:" (if such a summary is useful based on the discussion).
+  * If the community rejects the issue as an erratum, the issue should be closed.
+  * Each errata may be labelled as “`Editorial`”; editorial errata are listed separately from the substantial ones. By default, an issue is considered substantive unless indicated otherwise.
+  * ALL substantive errata are generally expected to have corresponding test(s) (such as a pull request in [web-platforms-tests](https://github.com/web-platforms-tests/wpt)), either in the form of new tests or modifications to existing tests, or must include the rationale for why test updates are not required for the erratum.
```
Comment (riccardoAlbertoni) [inline errata/README.md]:
```suggestion
```

File: errata/index.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        <dd><span id="number"></span></dd>
+        <dt>Link to all errata:</dt>
+        <dd><span id="errata_link"></span></dd>
+      </dl>
+
+      <section data-notoc>
+        <h1>How to Submit an Erratum?</h1>
+        <p>Errata are introduced and stored in the <a class="done" href="https://github.com/w3c/dxwg/issues/">issue list of the group‘s GitHub repository</a>. The workflow to add a new erratum is as follows:</p>
+        <ul>
+           <li>An issue is raised for a possible erratum. The label of the issue SHOULD be set to “<code>ErratumRaised</code>”. It SHOULD also include the label corresponding to the document on which the erratum is raised, e.g., “<code class="done">dcat</code>” or “<code class="todo">@@@@</code>”. (Please, consult the <a class="done" href="https://github.com/w3c/dxwg/labels">list of available labels</a>.) It is o.k. for an erratum to have several labels. In some, exceptional, cases, i.e., when the erratum is very general, it is also acceptable not to have a reference to a document.</li>
```
Comment (riccardoAlbertoni) [inline errata/index.html]: At the moment, we need it for DCAT only. We can update with other recs as soon as they are published.
  
```suggestion
           <li>An issue is raised for a possible erratum. The label of the issue SHOULD be set to “<code>ErratumRaised</code>”. It SHOULD also include the label corresponding to the document on which the erratum is raised, i.e,, “<code class="done">dcat</code>”. (Please, consult the <a class="done" href="https://github.com/w3c/dxwg/labels">list of available labels</a>.) It is o.k. for an erratum to have several labels. In some, exceptional, cases, i.e., when the erratum is very general, it is also acceptable not to have a reference to a document.</li>
```

File: errata/index.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+           <li>Issues labeled as “<code>Editorial</code>” are displayed separately, to make it easier to differentiate editorial errata from substantial ones.</li>
+          <li>The community discusses the issue. If it is accepted as a genuine erratum, the label “<code>Errata</code>” is added to the entry and the “<code>ErratumRaised</code>” label should be removed. Additionally, a new comment on the issue MAY be added, beginning with the word "Summary:" (if such a summary is useful based on the discussion).</li>
+          <li>If the community rejects the issue as an erratum, the issue should be closed.</li>
+          <li>Each errata may be labelled as “<code>Editorial</code>”; editorial errata are listed separately from the substantial ones.</li>
+          <li class="todo">ALL substantive errata are generally expected to have corresponding test(s) (such as a pull request in <a href='https://github.com/web-platforms-tests/wpt'>web-platforms-tests</a>), either in the form of new tests or modifications to existing tests, or must include the rationale for why test updates are not required for the erratum.</li>
```
Comment (riccardoAlbertoni) [inline errata/index.html:44]: I would delete the entire part related to tests. I do not think we need it for DCAT.
```suggestion
  
```

File: errata/test.html [hunk 4]
```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
```
Comment (riccardoAlbertoni) [inline errata/test.html]: Do we need the file "errata/test.html"? It seems we can avoid providing it. Am I missing something?

File: errata/README.md (hunk 1 above)
Comment (andrea-perego) [reply errata/README.md]: Thanks, @riccardoAlbertoni . I kept this file just temporarily, but it is actually not worth keeping it, at least in its current form. 

I think we can drop all the content and replace it with just a statement saying that the errata page has been created based on etc.

File: errata/index.html (hunk 3 above)
Comment (andrea-perego) [reply errata/index.html:44]: Agreed.

File: errata/test.html (hunk 4 above)
Comment (andrea-perego) [reply errata/test.html]: No, we can remove it.

File: errata/index.html (hunk 3 above)
Comment (andrea-perego) [reply errata/index.html:44]: I would however just comment it for the moment, instead of deleting that line.

File: errata/README.md (hunk 1 above)
Comment (andrea-perego) [reply errata/README.md]: Just done.

Are you happy with this revision, @riccardoAlbertoni ?

File: errata/index.html (hunk 3 above)
Comment (andrea-perego) [reply errata/index.html:44]: Just done.

Are you happy with this revision, @riccardoAlbertoni ?

File: errata/test.html (hunk 4 above)
Comment (andrea-perego) [reply errata/test.html]: Done.

File: errata/README.md (hunk 1 above)
Comment (riccardoAlbertoni) [reply errata/README.md]: Fine to me.

File: errata/index.html (hunk 3 above)
Comment (riccardoAlbertoni) [reply errata/index.html:44]: Perfect!

**Comment by riccardoAlbertoni** [comment]
Thank Andrea, I like the page you prepared. I only wonder if we need to keep the file LICENSE in the directory to meet the term of the license.

**Comment by andrea-perego** [comment]
@riccardoAlbertoni said:

> Thank Andrea, I like the page you prepared. I only wonder if we need to keep the file LICENSE in the directory to meet the term of the license.

I don't think so. My understanding is that that LICENSE file is related to the content of the original repo (display_errata). Also, I haven't found links from the errata document to the LICENSE file.

**Comment by riccardoAlbertoni** [comment]
> @riccardoAlbertoni said:
> 
> > Thank Andrea, I like the page you prepared. I only wonder if we need to keep the file LICENSE in the directory to meet the term of the license.
> 
> I don't think so. My understanding is that that LICENSE file is related to the content of the original repo (display_errata). Also, I haven't found links from the errata document to the LICENSE file.

Ok, Then  I would suggest marking this PR as "ready for review" and asking the editors to review it, So that, we can get it merged by the beginning of next week. What do you think?

**Comment by andrea-perego** [comment]
+1 from me.

**Review by riccardoAlbertoni — APPROVED** [review]

**Review by agbeltran — APPROVED** [review]

**Comment by riccardoAlbertoni** [comment]
also related to #545

**Pull request merged**